content/docs/command-reference/exp/save.md

```diff
@@ -0,0 +1,163 @@
+
+## Synopsis
+
+```usage
+usage: dvc exp save [-h] [-q | -v] [-f]
+                   [--json] [-n <name>]
+                   [-I path]
+```
+
+## Description
+
+Provides a way to save the current status of your <abbr>project</abbr> as an
+<abbr>experiment</abbr> without polluting it with unnecessary commits, branches,
+directories, etc.
+
+Only files tracked by either Git or DVC are saved to the experiment. Use the
+`--include-untracked` (`-I`) option to explicitly include any untracked files in
+the experiment.
+
+`dvc exp save` creates <abbr>experiments</abbr> just like
+[`dvc exp run`](/doc/command-reference/exp/run), but does not require
+reproduction or setup of [pipeline stages](/doc/start/data-pipelines), making it
+possible to quickly start tracking, [comparing] and [persisting] experiments.
+
+[comparing]: /doc/user-guide/experiment-management/comparing-experiments
+[persisting]: /doc/user-guide/experiment-management/persisting-experiments
+
+## Options
+
+- `-n <name>`, `--name <name>` - specify a [unique name] for this experiment. A
+  default one will be generated otherwise, such as `exp-f80g4` (based on the
+  experiment's hash).
+
+- `-I <path>`, `--include-untracked <path>` - specify untracked file(s) to be
+  included in the saved experiment. Multiple files can be specified.
+
+- `-f`, `--force` - overwrite the experiment if an experiment with the same name
+  already exists.
+
+- `-h`, `--help` - prints the usage/help message, and exits.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if all
+  stages are up to date or if all stages are successfully executed, otherwise
+  exit with 1. The command defined in the stage is free to write output
+  regardless of this flag.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+[unique name]:
+  https://dvc.org/doc/user-guide/experiment-management/experiments-overview#how-does-dvc-track-experiments
+
+## Examples
+
+<admon type="info">
+
+This example is based on [our Get Started], where you can find the actual source
+code.
+
+[our get started](/doc/start/experiment-management/experiments)
+
+</admon>
+
+Let's say we have modified our repo by adding new data (`data/new.xml`), new
+code (`src/extratrees.py`), as well as modifying the training script
+(`src/train.py`). After running `dvc add data/new.xml`, we have the following
+status in the repo:
+
+```cli
+$ git status
+On branch main
+Your branch is up to date with 'upstream/main'.
+
+Changes not staged for commit:
+  (use "git add <file>..." to update what will be committed)
+  (use "git restore <file>..." to discard changes in working directory)
+	modified:   data/.gitignore
+	modified:   src/train.py
+Untracked files:
+  (use "git add <file>..." to include in what will be committed)
+	data/new.xml.dvc
+	src/extratrees.py
+
+no changes added to commit (use "git add" and/or "git commit -a")
+```
+
+We can run `dvc repro` to reproduce the pipeline and inspect results, e.g. by
```

Comment:
(I asked this earlier, sorry for being repetitive.) Is this a happy path? `dvc repro` + `exp save` doesn't make much sense to me. I thought this is mainly for cases where you haven't setup a DVC pipeline.